catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/VisitableJsonElementImpl.java

```diff
@@ -138,6 +146,13 @@ private VisitableJsonElementImpl(
         new VisitableJsonElementImpl(FAKE_VALUE_OPERATOR, Collections.singletonList(value)));
   }
 
+  private static List<VisitableElement<?>> wrap(String property, String value, Double distance) {
+    return Arrays.asList(
```

Comment:
❓ Do we want this list to be immutable? If so then use `List.of` instead of `Arrays.asList`.